src/forum.ts

```diff
@@ -44,7 +46,9 @@ const handleAssistantConnection = async (event: APIGatewayProxyEvent, callback:
  * - Send to free/available assistants a new list of trainees awaiting help
  */
 const handleTraineeConnection = async (event: APIGatewayProxyEvent, callback: Callback<APIGatewayProxyResult>): Promise<void> => {
-  await peersTable.add({ connectionId: getConnectionId(event), status: 'TRAINEE_WAITING' });
+  const connectionId = getConnectionId(event);
```

Comment:
I don't understand what I am supposed to review...
`handleTraineeConnection` is modified... but it is only used by `handleConnection` which is not used anymore as a handler in `serverless.yml`.